src/index.js

```diff
@@ -0,0 +1,157 @@
+const city = document.getElementById('city_name')
+let inputCity = document.getElementById('city_input')
+
+function changeCity(event) {
+    if (event.key === 'Enter') {
+        city.textContent = inputCity.value;
+    }
+}
+inputCity.addEventListener('keydown', changeCity)
+inputCity.addEventListener('keydown', updateWeather)
+
+
+
+const increaseTempButton = document.getElementById('increase_temp')
+const decreaseTempButton = document.getElementById('decrease_temp')
+
+
+function increaseTemp() {
+    const temperature = document.getElementById('temp_no');
+    let current_temp = Number(temperature.textContent)
+
+    temperature.textContent = current_temp + 1;
+    tempColor(current_temp + 1);
+    changeLandscape(current_temp + 1)
+}
+
+increaseTempButton.addEventListener('click', increaseTemp)
+
+function decreaseTemp() {       
+    const temperature = document.getElementById('temp_no');
+    let current_temp = Number(temperature.textContent)
+
+    temperature.textContent = current_temp - 1;
+    tempColor(current_temp - 1);
+    changeLandscape(current_temp + 1)
+}
+decreaseTempButton.addEventListener('click', decreaseTemp)
+
+function changeLandscape(temp) {
+    let landscape = document.getElementById('fashion_landscape');
+    if (temp < 60) {
+        landscape.src = "assets/cold_outfit.jpeg"
+    } else if (temp >= 60 && temp < 70) {
+        landscape.src = "assets/60_degree_outfit.jpeg"
+    } else if (temp >= 70 && temp < 80) {
+        landscape.src = "assets/cardigan_and_jeans.jpeg"
+    } else {
+        landscape.src = "assets/80_degree_outfit.jpeg"
+    }
+
+}
+
+function tempColor(temp) {
+    let temperature = document.getElementById('temp');
+    if (temp <= 49) {
+        temperature.className = 'freezing';
+    } else if (temp > 49 && temp < 60) {
+        temperature.className = 'cold';
+    } else if (temp >= 60 && temp < 70) {
+        temperature.className = 'chilly';
+    } else if (temp >= 70 && temp < 80) {
+        temperature.className = 'hot';
+    } else {
+        temperature.className = 'scorching';
+    }
+}
+
+let skySelector = document.getElementById('sky_selector');
+let skyImage = document.getElementById('pinterest-sky');
+
+skySelector.addEventListener('change', changeSkyImage);
+
+function changeSkyImage() {
+    let selectedSky = skySelector.value;
+    switch(selectedSky) {
+        case 'skySunny':
+            skyImage.src = 'assets/skies/sunny_sky.jpeg';
+            break;
+        
+        case 'skyCloudy':
+            skyImage.src = 'assets/skies/cloudy_sky.jpeg';
+            break;
+
+        case 'skyRainy':
+            skyImage.src = 'assets/skies/rainy_sky.jpeg';
+            break;
+
+        case 'skySnowy':
+            skyImage.src = 'assets/skies/snowy_sky.jpeg';
+            break;
+
+        case 'skyStarry':
+            skyImage.src = 'assets/skies/starry_sky_blue.jpeg';
+            break;
```

Comment:
Note that there are some formatting issues for the `Starry` and `Snowy` options where the section header `The Sky Today` and the dropdown menu for sky selection get pushed out of the visible area of the box due to the images being too large. This makes a page refresh required to get the selection box back.

For things like this, it's really good to restrict the maximum size of images, make all images the same size, and/or make the position of the header and dropdown menu fixed so they can't disappear.